doc/pages/source/documentation/index.md

```diff
@@ -268,6 +270,38 @@ def self.trigger_task(async, action, *args, &block)
 end
 ```
 
+#### Scheduling
+
+Scheduling an action means setting it up to be triggered at set time in future.
+Any action can be scheduled by calling:
+
+```ruby
+world_instance.schedule(AnAction, start_at, start_before, *args)
```

Comment:
Two things to suggest:
* using the scheduling options as hash: if we introduce another option, we don't break the backward compatibility with the users' code
* Use specific values instead of the variables in the example, so that the users know imediately, whether Time object or String in some format is expected

 `world_instance.schedule(AnAction, {start_at: Time.now + 360, start_before: Time.now + 400}, *args)`